### DISCUSSION TITLE
New releases view doesn't show the release date

### DISCUSSION THREAD

**Discussion by realsimix** [community/community#6238] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

As important as the release version is also the release date for me.

This was prominently shown with the old version but now it is gone.

Regards,
Simon

**Comment by MylesBorins** [comment, marked as answer]
The date is there, in the exact same format as before

list view:

![Screen Shot 2021-10-12 at 6 47 29 PM](https://user-images.githubusercontent.com/498775/137039108-c5a9987b-cb0a-4b48-a3ca-4b503db32865.png)

release landing page
![Screen Shot 2021-10-12 at 6 47 10 PM](https://user-images.githubusercontent.com/498775/137039121-2cef4492-77ce-44b3-b861-a60afef6cd68.png)

**Reply by realsimix** [reply]
I see, my example is here

https://github.com/akheron/jansson/releases

With the old version it shows that version 2.14 was `akheron released this on 9 Sep`.

With the new version it only shows `last month` on the left side of the page.

Thanks,
Simon

**Reply by MylesBorins** [reply]
InterestingThanks for taking the extra time to show the difference here. I've opened a bug with engineering to bring back the old behavior

**Reply by MylesBorins** [reply]
As a quick update... we are going to start working on a fix for this in the next week or two... it will be a bit more work than it seems on the surface though as we can't reuse the existing components

**Reply by maricn** [reply]
at least a mouseover hover popup showing the absolute date would be helpful..

**Reply by impeeza** [reply]
I did think was something wrong on my side.

**Reply by MylesBorins** [reply]
This should be fixed now. After 30 days we show the full text of the date

**Reply by MylesBorins** [reply]
We did a bit more digging and just realized that we have a rendering bug in safari. I've created a ticket with engineering and we are going to prioritize a fix

**Reply by Mantas-2155X** [reply]
I am also experiencing the issue of release date not being visible in the list view. The date once I enter a release is visible.
Tested with my main profile and an empty profile without any extensions.

EDIT: Not visible with Firefox on Android in both mobile view and desktop view as well. Chrome is fine.

Firefox 93 on Linux
![image](https://i.ibb.co/cQXkMWJ/2021-10-23-15-44.png)

**Reply by caseif** [reply]
I can also confirm Firefox 93 on Linux being affected. Latest Chromium and Edge work render the datetime properly on the same system.

**Reply by Detruire** [reply]
The bug is also affecting Firefox 94 (b9, Windows and Linux).

**Reply by maricn** [reply]
> This should be fixed now. After 30 days we show the full text of the date

@MylesBorins 

I prefer not to wait for 30 days to be able to see absolute time.

If I need to correlate a bug in production, it's really not helpful to see "26 days ago" - and then I need to think whether those are full days, whether today is also taken into that calculation, etc. These are not purely logical questions but also culturally/language dependent and they always necessitate a cognitive effort (at least for me).

If having a "smart" relative+absolute release time shown is deemed best solution from your end, I kindly ask you to take into consideration adding an unconditional (on 30d+) popup with exact time on mouse hover.

Additionally, it's unfortunate that the time of release is not visible in the "Releases" overview, but only once you navigate into some particular release itself.

**Reply by MylesBorins** [reply]
As an update we have fixed the bug in FF + Safari and the date should show for all platforms now 🎉 

@maricn we tend to try and make consistent design decisions here at GitHub, in particular the component to decide how many days before showing the full date is used consistently in a couple places across the platform. I can poke a few folks in design about making a change to the default but we would not be looking to make a change that would only be for releases

**Reply by maricn** [reply]
Ah, this is embarrassing, i disabled popups on my browser, so the on-mouse-hover text wouldn't appear. Now that I have an option to see the full time, I'm fine.

Sorry for the inconvenience caused by my mistake. Btw, I completely understand the decision to have consistent design. Thanks for your support.

**Comment by przemolb** [comment]
I confirm it is still affecting FF 93.0.
![image](https://user-images.githubusercontent.com/5048651/138702414-c9acdea2-3d3f-4186-bdcf-e758b7e9f9ff.png)

**Reply by MylesBorins** [reply]
Thsi should be fixed now

**Comment by impeeza** [comment]
I can confirm is working on FireFox 93.0 64 bits on Windows.

**Comment by zsolt-sb** [comment]
The hover time for this takes very long.   I always thought the implementation should be the reverse.  Get the datetime automatically and hover if you want a human approximation.
